astropy/stats/jackknife.py

```diff
@@ -0,0 +1,194 @@
+# Licensed under a 3-clause BSD style license - see LICENSE.rst
+
+from __future__ import (absolute_import, division, print_function,
+                        unicode_literals)
+import numpy as np
+from scipy.special import erfinv
```

Comment:
I'm not sure, it got restructured since, and you were right that it shouldn't been in the very middle of a function.